core/include/core/ippu.h

```diff
@@ -10,6 +10,10 @@ class IPpu {
 public:
     virtual ~IPpu() = default;
 
+    virtual uint8_t read_byte(uint16_t addr) const = 0;
+
```

Comment:
I think it's nicer when functions are grouped in some way, so maybe remove this blank line? I don't know what you guys think, but I think having one blank line per line of code is really hard to parse.

Reply:
I can, without any problem, read code that is grouped OR have one blank line per line of code. That is how good I'm. But to make life easier for you Robin I'll change this and start think about my blank lines.